### DISCUSSION TITLE
Prohibited Browser Configuration error

### DISCUSSION THREAD

**Discussion by ldh007bucknell** [SafeExamBrowser/seb-win-refactoring#682] [General]
We were using Safe Exam Browser in a Moodle quiz.  One of the students was using a Dell laptop with a single display only.  He was able to launch Safe Exam Browser (SEB) - Windows version 3.5.0 - and begin his Moodle exam.  

Midway through the exam, he got a "Prohibited Display Configuration" error with two radio button options:  ignore the error, or save your progress and restart the computer.  Either option required a password, but the exam password was rejected each time.

To solve the problem, we had the student log in to Moodle using a different computer.

Other students were using the same exact Dell laptop and didn't see that error.

What password was the SEB expecting to ignore the configuration error (and where do you set that password)?  What is the solution when a student sees that error?

We came prepared (bringing multiple department-owned laptops to the exam site), but we would prefer to avoid this error in the future.

Leslie Harris
Bucknell University

**Comment by dbuechel** [comment]
> What password was the SEB expecting to ignore the configuration error (and where do you set that password)? What is the solution when a student sees that error?

An invigilator will then need to enter the unlock password, see https://safeexambrowser.org/windows/win_usermanual_en.html#GeneralPane. If it appears to be a wrong positive detection, the invigilator can choose to ignore the error and continue with the exam or otherwise terminate SEB.

To determine what exactly happened and why, we'd need the log files of the affect session from the student's computer.
